### DISCUSSION TITLE
Ignoring all the path after a certain component.

### DISCUSSION THREAD

**Discussion by oliverfoggin** [pointfreeco/swift-url-routing#85] [Q&A]
When parsing a URL is it possible to parse a wildcard at the end of the path?

i.e. with this...

```
// GET /app/:ID/*
Route(.case(OurEnum.case(id:)) {
  Path {
    "app"
    Parse(.string)
    // what goes here?
  }
}
```

The wildcard path might be anything and might be any length.

Also... it might not have anything on. So all of these are valid...

- /app/123
- /app/123/foo
- /app/123/bar
- /app/123/foo/bar
- /app/123/bar/foo
- /app/123/a/b/c/d/e/f/g/h/i/j/k/...

And should parse into `OurEnum.case(id: "123")`

**Comment by tfe** [comment]
@oliverfoggin Did you ever find a solution to this?
